### PR TITLE
To Do List : Interactive list 2nd part

### PR DESCRIPTION
**Pull request by aboongm**
- Add a new JavaScript file viz checkbox.js and import it as a module:
- it has methods completed() method related to the status updates (completed: true / false) and and clearCompletedTasks().
- Add event bubbling by calling completed() inside taskList event listener.
- Update items object's value for completed key upon user actions.
- Implement a function clearCompletedTasks() for the "Clear all completed" button using filter() method.
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/styles/style.css [hunk 1]
```diff
@@ -43,3 +43,7 @@ input:focus {
 span:focus {
   outline: none;
 }
+
+/* input[type='checkbox']:checked + label.strikethrough {
+  text-decoration: line-through;
+} */
```
Comment (Meltrust) [inline src/styles/style.css]: - [ ] Please, remove this old piece of commented code so the repo gets cleaner 💪

File: src/module/constElements.js [hunk 2]
```diff
@@ -2,17 +2,11 @@ const refreshTask = document.querySelector('#refresh');
 const taskInput = document.querySelector('#task');
 const submitInput = document.querySelector('#arrow-submit');
 const taskList = document.querySelector('#taskList');
-const clearCompletedTasks = document.querySelector('#clear-completed');
+// const clearCompletedTasks = document.querySelector('#clear-completed');
```
Comment (Meltrust) [inline src/module/constElements.js]: - [ ] Please, remove this old piece of commented code so the repo gets cleaner 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @aboongm,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice code organization ✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @aboongm , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**